Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/ClearFee.yaml

```diff
@@ -0,0 +1,30 @@
+imports:
+  ServiceNames: Dashboard.ProviderPlatform.Driver
+  Currency: Kernel.Types.Common
+  HighPrecMoney: Kernel.Types.Common
+
+module: ClearFee
```

Comment:
I think it should be `Driver.yaml`
then path will be generated like `driver/clearFee`